lib/Dist/Zilla/Plugin/Alien.pm

```diff
@@ -500,19 +500,27 @@ use strict;
 use warnings;
 use File::ShareDir ':ALL';
 use Path::Class;
+use {{ $mod }}::ConfigData;
 
-my $abs = file(dist_dir('{{ $dist->name }}'),'bin','{{ $bin }}')->cleanup->absolute;
+if(Alien::ffmpeg::ConfigData->config('install_type') ne 'system') {
+	my $abs = file(dist_dir('{{ $dist->name }}'),'bin','{{ $bin }}')->cleanup->absolute;
+} else {
+	$abs = "/usr/bin/{{ $bin }}";
```

Comment:
There is no way that we know the system executable will be installed in `/usr/bin`, or even if that directory exists.  Better would be not to install the wrapper for system install, or maybe use `File::Which` to find the next executable in the path with the appropriate name.

Reply:
Did that, thanks.